demos/grouped_mask_prostate_longitudinal/README.md

```diff
@@ -36,7 +36,7 @@ python demos/grouped_mask_prostate_longitudinal/demo_data.py
   [`dataset` section](./grouped_mask_prostate_longitudinal_dataset0.yaml) and the
   [`train` section](./grouped_mask_prostate_longitudinal_train.yaml), which can be
   specified in
-  [seperate yaml files](https://deepregnet.github.io/DeepReg/#/tutorial_experiment?id=cross-validation);
+  [separate yaml files](https://deepreg.readthedocs.io/en/latest/tutorial/cross_val.html);
```

Comment:
> Just one minor comments. There are a couple of dead links in the readthedocs files (dataset loader and configuration file). Not specifically linked to this issue, but wanted to flag it in case nobody has noticed.

Thanks! will fix these in next PR #308